addons/web/static/src/views/list/list_renderer.scss

```
@@ -409,37 +445,47 @@
         }
     }
 
-    /**
-    Because of border-collapse, border top and border bottom are 'merged'.
+    @include media-only(screen) {
+        /**
+        Because of border-collapse, border top and border bottom are 'merged'.
 
-    * cell border overlaps row border
-    * when there is a border collapse the first cell (e.g. A1) take over the following cell (e.g. B1 or A2)
-    ┌──┬──┐
-    │A1│B1│
-    ├──┼──┤
-    │A2│B2│
-    └──┴──┘
+        * cell border overlaps row border
+        * when there is a border collapse the first cell (e.g. A1) take over the following cell (e.g. B1 or A2)
+        ┌──┬──┐
+        │A1│B1│
+        ├──┼──┤
+        │A2│B2│
+        └──┴──┘
 
-    So we apply some rules to have the style wanted:
-    */
-    // 1. we set the global separator (gray line) of the table on the row (bottom) and not on the cell (bottom)
-    .o_data_row {
-        border-bottom-width: 1px;
+        So we apply some rules to have the style wanted:
+        */
+        // 1. we set the global separator (gray line) of the table on the row (bottom) and not on the cell (bottom)
+        .o_data_row {
+            border-bottom-width: 1px;
 
-        &:not(.o_selected_row):not(.o_data_row_selected):focus-within > * {
-            $-focus-bg: rgba(var(--#{$variable-prefix}emphasis-color-rgb), #{$table-hover-bg-factor * 2});
+            &:not(.o_selected_row):not(.o_data_row_selected):focus-within > * {
+                $-focus-bg: rgba(var(--#{$variable-prefix}emphasis-color-rgb), #{$table-hover-bg-factor * 2});
 
-            --#{$variable-prefix}table-accent-bg: #{$-focus-bg};
+                --#{$variable-prefix}table-accent-bg: #{$-focus-bg};
+            }
         }
-    }
 
-    // 2. we remove the bottom border (added by BS5) on all cells (this force to show the top border of selected cell)
-    .o_data_row > .o_data_cell {
-        border-bottom-width: 0;
+        // 2. we remove the bottom border (added by BS5) on all cells (this force to show the top border of selected cell)
+        .o_data_row > .o_data_cell {
+            border-bottom-width: 0;
+        }
+        // 3. we add only a bottom border to selected cells (this force to show the bottom border of the selected cell)
+        .o_data_row.o_data_row_selected > .o_data_cell {
+            border-bottom-width: 1px;
+        }
     }
-    // 3. we add only a bottom border to selected cells (this force to show the bottom border of the selected cell)
-    .o_data_row.o_data_row_selected > .o_data_cell {
-        border-bottom-width: 1px;
+
+    @include media-only(print) {
+        .o_data_cell {
+            [type="action"] {
+                display: none;
+            }
+        }
```

Nitpicking: could we simplify this ?
```suggestion
        .o_data_cell [type="action"] {
              display: none;
        }
```